### PR TITLE
fzf: clean post_install()

### PR DESCRIPTION
**Pull request by FollieHiyuki**
Add fzf-tmux man page and more vim doc.
Change the installed files to a new location

#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by ericonr** [comment]
Is changing the install location worth it? Someone symlink from that location will have broken symlinks after an update.

**Comment by FollieHiyuki** [comment]
Not really :) I just feel like we should not put random things inside `/usr/share/doc`.
If symlink problem is concerned we can have a INSTALL.msg. Probably?

File: srcpkgs/fzf/template [hunk 1]
```diff
@@ -18,14 +18,17 @@ post_install() {
...
 	vinstall plugin/fzf.vim 644 usr/share/nvim/runtime/plugin
-	vinstall shell/completion.bash 644 usr/share/doc/fzf
-	vinstall shell/completion.zsh 644 usr/share/doc/fzf
-	vinstall shell/key-bindings.zsh 644 usr/share/doc/fzf
-	vinstall shell/key-bindings.bash 644 usr/share/doc/fzf
-	vinstall shell/key-bindings.fish 644 usr/share/doc/fzf
+	vinstall doc/fzf.txt 644 usr/share/vim/vimfiles/doc
+	vinstall doc/fzf.txt 644 usr/share/nvim/runtime/doc
+
+	vinstall shell/completion.bash 644 usr/share/fzf
```
[the remark this answers is not in the document]
Comment (FollieHiyuki) [reply srcpkgs/fzf/template:29]: I use ** quite a lot and i find it annoying whenever I accidentally trigger its completion of `fzf`.
So yeah should be optional.

File: srcpkgs/fzf/template [hunk 2]
```diff
@@ -18,14 +18,17 @@ post_install() {
...
 
-	sed -i -e 's#source ~/\.fzf\.bash; ##' shell/key-bindings.bash
 	vinstall plugin/fzf.vim 644 usr/share/vim/vimfiles/plugin
 	vinstall plugin/fzf.vim 644 usr/share/nvim/runtime/plugin
-	vinstall shell/completion.bash 644 usr/share/doc/fzf
-	vinstall shell/completion.zsh 644 usr/share/doc/fzf
-	vinstall shell/key-bindings.zsh 644 usr/share/doc/fzf
-	vinstall shell/key-bindings.bash 644 usr/share/doc/fzf
-	vinstall shell/key-bindings.fish 644 usr/share/doc/fzf
+	vinstall doc/fzf.txt 644 usr/share/vim/vimfiles/doc
```
[the remark this answers is not in the document]
Comment (FollieHiyuki) [reply srcpkgs/fzf/template:26]: This is vim plugin doc for `fzf.vim`. It has to be there

File: srcpkgs/fzf/template [hunk 3]
```diff
@@ -18,14 +18,17 @@ post_install() {
...
+	vinstall doc/fzf.txt 644 usr/share/nvim/runtime/doc
+
+	vinstall shell/completion.bash 644 usr/share/fzf
+	vinstall shell/completion.zsh 644 usr/share/fzf
+	vinstall shell/key-bindings.zsh 644 usr/share/fzf
+	vinstall shell/key-bindings.bash 644 usr/share/fzf
+	vinstall shell/key-bindings.fish 644 usr/share/fzf
```
[the remark this answers is not in the document]
Comment (FollieHiyuki) [reply srcpkgs/fzf/template:33]: That is an option. But these files don't feel like configuration for me though

**Pull request closed without merging**